aces/joint_deconvolution/feather_all_spws.py

```diff
@@ -0,0 +1,260 @@
+import os
+import glob
+import shutil
+import pandas as pd
+from tqdm import tqdm
+from pathlib import Path
+from astropy.table import Table
+from casatasks import importfits, exportfits, imhead, imreframe, imtrans, feather, imsmooth
+
+
+def check_files_exist(file_names):
+    return all(file_name is not None and Path(file_name).exists() for file_name in file_names)
+
+
+def get_file(filename):
+    """
+    Retrieve the file matching the provided pattern using glob.
+    If multiple files are found, it sorts the files and returns the first one
+    (this should not be necessary once we remove duplicate images).
+    """
+    files = glob.glob(filename)
+    if len(files) == 0:
+        print(f"[INFO] No files matching '{filename}' were found.")
+        return None
```

Comment:
```suggestion
        return
```
`return None` is redundant